rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -128,8 +114,7 @@ def _getBondOffset(self, p1, p2):
     return perp, offsetX, offsetY
 
   def _getOffsetBondPts(self, p1, p2, offsetX, offsetY, lenFrac=None):
-    if not lenFrac:
-      lenFrac = self.drawingOptions.dblBondLengthFrac
+    lenFrac = lenFrac or self.drawingOptions.dblBondLengthFrac
```

Comment:
It's a fairly common idiom I use as well.  It can byte you though if 0.0 values are allowed which isn't the case here though in either version of the code.